homeassistant/components/yolink/services.py

```diff
@@ -19,6 +19,11 @@
 
 SERVICE_PLAY_ON_SPEAKER_HUB = "play_on_speaker_hub"
 
+_SPEAKER_HUB_PLAY_CALL_OPTIONAL_ATTRS = (
+    (ATTR_VOLUME, lambda x: x),
+    (ATTR_TONE, lambda x: x.capitalize()),
```

Comment:
Why do we now capitalize the tone?

Reply:
It was capitalized before. I just moved the code.